dfc/keepalivetracker.go

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Is this header necessary?

Reply:
you mean the copy right?
every files has one.
it is not enough, i think later if doc is open sourced, we have to add more text approved by nvidia's legal to describe the license terms